lib/sycamore/sycamore/writers/file_writer.py

```diff
@@ -100,6 +101,18 @@ def elements_to_bytes(doc: Document) -> bytes:
     return out.getvalue().encode("utf-8")
 
 
+def document_to_bytes(doc: Document) -> bytes:
+    """
+    Returns a UTF-8 encoded json string of the document.  Adds newline.
+    Beware this will try to interpret binary_representation as UTF-8.
```

Comment:
Is this ever what we want? Should we just skip the binary_representation entirely?

Reply:
Looking at JSONEncodeWithUserDict, it doesn't seem possible to know what "bytes" object we're looking at.  I figured it was safest not to mess with that.